File: backend/app/models/booking.py
from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
from app.database import Base

class Booking(Base):
    __tablename__ = "bookings"

    id = Column(Integer, primary_key=True, index=True)
    event_id = Column(Integer, ForeignKey("events.id"), nullable=False)
    venue_id = Column(Integer, ForeignKey("venues.id"), nullable=False)
    start_time = Column(DateTime, nullable=False)
    end_time = Column(DateTime, nullable=False)

File: frontend/src/types/booking.ts
export interface Booking {
  id: number;
  event_id: number;
  venue_id: number;
  start_time: string;
  end_time: string;
}

export interface BookingCreate {
  event_id: number;
  venue_id: number;
  start_time: string;
  end_time: string;
}